Makefile.in

```diff
@@ -3,7 +3,7 @@ CC=cc
 
 all: src/Makefile.in
 	mkdir -p bin/@GAPARCH@;
-	sed -e "s/@DATE@/`date`/g" src/Makefile.in >src/Makefile
+	sed -e "s/@DATE@/`date -u -r CHANGES`/g" src/Makefile.in >src/Makefile
```

Comment:
This uses non-POSIX extensions of GNU sed, and won't work on BSD / macOS

Reply:
Err, I shpuld have said: it uses a GNU extension of date (not sed), namley -r, which is not in POSIX. But it seems BSD supports it after all, though I have not checked that the semantics match.

Still, I'd prefer to get rid of this substitution.